src/actions/RespondActions.js

```diff
@@ -0,0 +1,16 @@
+import petitionRepository from 'services/api/repositories/petition';
+
+import {
+  requestPetition,
+  receivePetition
+} from 'actions/PetitionActions';
+
+export function fetchPetition (responseToken) {
```

Comment:
`fetchPetitionByResponseToken`?

Besides this, is it ok to separate this from the other `PetitionActions` but reuse the `requestPetition` and `receivePetition` actions here?